src/sentry/ingest/ingest_consumer.py

```diff
@@ -384,9 +385,29 @@ def process_individual_attachment(message, projects) -> None:
         logger.exception("invalid individual attachment type: %s", attachment.type)
         return
 
+    new_attachment = attachment
+    # We need the event to get the proguard file
+    # TODO: What if we don't have the event? Do we drop the attachment?
+    if (
+        attachment.type == "event.view_hierarchy"
+        and event is not None
+        and has_proguard_file(event.data)
```

Comment:
Flutter events are similar to iOS, it uses the debug images for symbolication and not the proguard file, so I believe we need an `or` condition here?